include/openssl/ssl.h.in

```diff
@@ -410,6 +410,8 @@ typedef int (*SSL_async_callback_fn)(SSL *s, void *arg);
      * interoperability with CryptoPro CSP 3.x
      */
 # define SSL_OP_CRYPTOPRO_TLSEXT_BUG                     SSL_OP_BIT(31)
+    /* Enable kTLS TX zerocopy on Linux */
+# define SSL_OP_ENABLE_KTLS_TX_ZEROCOPY_SENDFILE         SSL_OP_BIT(32)
```

Comment:
Yay for 64bit options!

Reply:
And with the certificate compression code added, this needs to be bumped up to `SSL_OP_BIT(34)`.